### PR TITLE
[ID-3587]: add I18nLocaleProvider component with a storybook

### PR DESCRIPTION
**Pull request by jmiklevska**
- added I18nLocaleProvider component (that depends from skycell-ag/auth Auth) for correctly set current language for 'react-i18next' localization provider
- added story with example of usage (env variables is needed)

### CODE REVIEW

File: .storybook/i18n.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  },
+  resources: {
+    en: {
+      translation: {
+        "SomeLocalizedKey": "It's translated to English"
+      }
+    },
+    de: {
+        translation: {
+          "SomeLocalizedKey": "It should be some German text"
```
Comment (dimaPortyanka) [inline .storybook/i18n.js:22]: Es ist Deutsch :)

File: src/I18nLocaleProvider/I18nLocaleProvider.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    const {
+        user,
+    } = useAuth()
+    const {
+        i18n,
+    } = useTranslation()
+
+    useEffect(() => {
+        if (i18n.isInitialized) {
+            i18n.changeLanguage(user?.attributes?.locale)
```
Comment (dimaPortyanka) [inline src/I18nLocaleProvider/I18nLocaleProvider.js:28]: if no user and no attributes?
as i remember user should exist in any case or component wont be rendered

File: src/I18nLocaleProvider/I18nLocaleProvider.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        }
```
Comment (dimaPortyanka) [inline src/I18nLocaleProvider/I18nLocaleProvider.js:29]: what should be if it is not initialized?
don't we need to initialize it then

File: src/I18nLocaleProvider/I18nLocaleProvider.js (hunk 2 above)
Comment (jmiklevska) [reply src/I18nLocaleProvider/I18nLocaleProvider.js:28]: if component would be used outside of <Auth> than there would be no user
not sure if user always return [] for attributes if there no attributes in keycloak

File: src/I18nLocaleProvider/I18nLocaleProvider.js (hunk 3 above)
Comment (jmiklevska) [reply src/I18nLocaleProvider/I18nLocaleProvider.js:29]: if i18n or  react-i18next is not initialized there would be warning in console:
 
react-i18next: You will need to pass in an i18next instance by using initReactI18next'
 and on UI we will see not the text but localization keys.

File: src/I18nLocaleProvider/I18nLocaleProvider.js (hunk 2 above)
Comment (dimaPortyanka) [reply src/I18nLocaleProvider/I18nLocaleProvider.js:28]: is atributes and array?

File: src/I18nLocaleProvider/I18nLocaleProvider.js (hunk 3 above)
Comment (dimaPortyanka) [reply src/I18nLocaleProvider/I18nLocaleProvider.js:29]: so can we say that we expect that it is always initialized?

File: src/I18nLocaleProvider/I18nLocaleProvider.js (hunk 2 above)
Comment (jmiklevska) [reply src/I18nLocaleProvider/I18nLocaleProvider.js:28]: no, my bad - it's object

**Review by dimaPortyanka — APPROVED** [review]

**Pull request merged**